app/views/dashboard/_weather.html.erb

```diff
@@ -1,4 +1,4 @@
-<% if ((selected_user.organisation.present?) && (lat = selected_user.club.lat) && (lon = selected_user.club.lon)) %>
+<% if ((lat = selected_user.club.lat) && (lon = selected_user.club.lon)) %>
```

Comment:
Is there a scenario where someone would not belong to a club other than an admin?

If no club exists, this logic will not be true and simply won't display.

If multiple a user belongs to multiple club, this will select the first club in the query response. Possibly need to set a 'default' or 'preferred' club in this instance if we don't go down the path to require a selection of user, and subsequent club where multiple users at linked to the same email.

Reply:
> Is there a scenario where someone would not belong to a club other than an admin?

No, all Swapsea users must be in SurfGuard.
When we get to multi-club membership, the selected_user would still be just one club, but the member gets to choose which Club that is upon login. We'll cross that bridge soon :)

Reply:
Sounds good.